public/index.html

```diff
@@ -40,6 +40,6 @@
 </div>
 
 <div id="message-bar" class="swagger-ui-wrap">&nbsp;</div>
-<div id="swagger-ui-container" class="swagger-ui-wrap"></div>
+<div id="swagger-ui-container" class="swagger-ui-wrap" style="padding: 20px"></div>
```

Comment:
I know, I guess I should not have blindly copied this changes from Shelbys' repository. Let me fix this in strong-swagger-ui.